Ray.jl/test/task.jl

```diff
@@ -1,30 +1,27 @@
 @testset "Submit task" begin
-    oid1 = submit_task(length, ("hello",))
-    oid2 = submit_task(max, (0x00, 0xff))
```

Comment:
making these concurrent was intentional, that way we at least stress the path where there are >1 task in flight.  not the biggest deal though adn we shoudl probably be testing that separately/explicitly.

Reply:
Multiple tasks in flight should definitely be tested separately. I noticed the way we laid out the tests before it was easy to forget to update the suffix on `oid` or `result` and accidentally get a test pass. This just avoids those kinds of copy/paste issues

Reply:
https://github.com/beacon-biosignals/ray_core_worker_julia_jll.jl/issues/50